tests/openwisp2/sample_notifications/apps.py

```diff
@@ -22,3 +26,9 @@ def register_notification_types(self):
                 'email_subject': '[{site.name}] INFO: {notification.target} created',
             },
         )
+
+    def connect_recievers(self):
+        from openwisp_notifications.handlers import register_notification_observation
```

Comment:
is it necessary to import this here?

Reply:
Sorry I forget to answer this last time. 
Yes, it is necessary because when I was importing it at the top of the file it was giving error that apps not loaded.